disperser/dataapi/subgraph/api.go

```diff
@@ -209,6 +211,37 @@ func (a *api) QueryOperatorInfoByOperatorIdAtBlockNumber(ctx context.Context, op
 	return &query.Operator, nil
 }
 
+func (a *api) QueryOperatorEjectionsGteBlockTimestampByOperatorId(ctx context.Context, blockTimestamp uint64, operatorId string) ([]*OperatorEjection, error) {
+	var (
+		query     queryOperatorEjectedsByOperatorID
+		variables = map[string]any{
+			"blockTimestamp_gte": graphql.Int(blockTimestamp),
+			"operatorId":         graphql.String(operatorId),
+		}
+	)
+	err := a.operatorStateGql.Query(context.Background(), &query, variables)
+	if err != nil {
+		return nil, err
+	}
+
+	return query.OperatorEjections, nil
+}
+
+func (a *api) QueryOperatorEjectionsGteBlockTimestamp(ctx context.Context, blockTimestamp uint64) ([]*OperatorEjection, error) {
+	var (
+		query     queryOperatorEjectedsGteBlockTimestamp
+		variables = map[string]any{
+			"blockTimestamp_gte": graphql.Int(blockTimestamp),
+		}
+	)
+	err := a.operatorStateGql.Query(context.Background(), &query, variables)
```

Comment:
It may have a limit of how many items to return, if there are more ejections than that, it'll cut off (hence not returning e.g. 90 days worth of ejections)